samples/snippets/src/main/java/com/example/spanner/HubbleTransactionsCodeLab.java

```diff
@@ -368,15 +381,49 @@ private void reads(
       }
       if (isStrong) {
         ResultSet throwaway =
-            dbClient.singleUse().read("Message", keySetBuilder.build(), Arrays.asList("body"));
+            dbClient
+                .singleUse()
+                .read("Message", keySetBuilder.build(), Collections.singletonList("body"));
         while (throwaway.next()) {}
       } else {
         ResultSet throwaway =
             dbClient
                 .singleUse(TimestampBound.ofMaxStaleness(15, TimeUnit.SECONDS))
-                .read("Message", keySetBuilder.build(), Arrays.asList("body"));
+                .read("Message", keySetBuilder.build(), Collections.singletonList("body"));
         while (throwaway.next()) {}
       }
     }
+    System.out.println("Finished execution of reads()");
+  }
+
+  private void executeTasksInParallel(Runnable task, ExecutorService executorService) {
+    int numProcessors = ((ThreadPoolExecutor) executorService).getMaximumPoolSize();
```

Comment:
We should have this be a parameter to this function, and then compute this once, where we have other constants.

Reply:
Makes sense